app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CreateEventActivity.java

```diff
@@ -277,6 +296,8 @@ private void hideKeyboard() {
     private boolean handleOnBackPressed() {
         String title = titleView.getText().toString();
         String description = descriptionView.getText().toString();
+
+        // TODO: If the user is in edit mode, check whether any data was changed.
```

Comment:
Do you still want to do this?

Reply:
Again, a “do this at some point” comment.